src/app/Instance/InstancePage/InstancePage.tsx

```diff
@@ -198,12 +204,14 @@ const InstancePage = (): JSX.Element => {
           >
             <Tab
               eventKey={0}
+              ouiaId="processors-overview"
```

Comment:
```suggestion
              ouiaId="processors-tab"
```

Reply:
I think similar change ^ allows in cypress pseodo code just `cy.get('processors-tab').click()` vs `cy.get('tab').get('processors-overview').click()`

Reply:
@jomarko I was considered to call `cy.ouiaId("processors-overview", "PF4/TabButton").click();`. Based on OUIA specification the usage of the id together with the type (generated by PF) is the safe way to identify the element in the particular context. I tried to avoid the duplication of the type in the id like `Tab` in the identificator.

I have missed one generated ouiaId for the Tab fragment in this context. What do you think about `cy.ouiaId("instance-details", "PF4/Tabs").ouiaId("processors", ""PF4/TabButton").click();`. 

![image](https://user-images.githubusercontent.com/1536609/166641749-d7a8ff7e-a2f1-4624-a659-c8083b7fcf2c.png)

Reply:
@sjamboro oh, that makes a difference. Ok. yes I agree

`cy.ouiaId("instance-details", "PF4/Tabs").ouiaId("processors", ""PF4/TabButton").click();` is quite easy to understand.  It was not used in this PR for now, I understand it will be in separate jiras. 

Could you please go trough all my comments, maybe some are still relevant, some are for sure not with this discussion.